src/Strategy/Elixir/MixTree.hs

```diff
@@ -138,68 +166,60 @@ parseDepSCM = (try (try parseDepHex <|> parseDepSCMGit) <|> parseDepSCMLocalPath
 
     parseDepSCMGit :: Parser DepSCM
     parseDepSCMGit = do
-      uri <- chunk "http://" <|> chunk "https://" <|> chunk "git://"
-      addr <- takeWhileP (Just "git address") (\c -> c /= ' ' && c /= ')')
+      uriScheme <- chunk "http://" <|> chunk "https://" <|> chunk "git://"
+      uriRest <- takeWhileP (Just "git address") (\c -> c /= ' ' && c /= ')')
 
       reference <- optional . try $ do
         _ <- chunk " - "
-        git_ref <- T.pack <$> some (alphaNumChar <|> char '.')
-        return (git_ref)
+        Text.pack <$> some (alphaNumChar <|> char '.')
 
-      pure (Git (uri <> addr) (reference))
+      pure $ Git (uriScheme <> uriRest) reference
 
-    parseDepSCMLocalPath :: Parser DepSCM
-    parseDepSCMLocalPath = do
-      path <- takeWhileP (Just "local path") (/= ')')
-      pure (LocalPath path)
+    parseDepSCMOther :: Parser DepSCM
+    parseDepSCMOther = Other <$> takeWhileP (Just "OtherSCM") (/= ')')
 
 type Parser = Parsec Void Text
 
 -- | True if a version is not resolved in `MixDepResolved`, otherwise False.
 -- This can happen, if dependencies are not retrieved or compiled.
-missingResolvedVersions :: M.Map PackageName MixDepResolved -> Bool
-missingResolvedVersions mdr = any isNothing (depResolvedVersion <$> map snd (M.toList mdr))
+missingResolvedVersions :: Map PackageName MixDepResolved -> Bool
+missingResolvedVersions mdr = any isNothing (depResolvedVersion <$> map snd (Map.toList mdr))
 
 -- Parses `mix deps` output
-mixDepsCmdOutputParser :: Parser (M.Map PackageName MixDepResolved)
-mixDepsCmdOutputParser = M.fromList <$> (parseDep `sepBy` char '*') <* eof
+mixDepsCmdOutputParser :: Parser (Map PackageName MixDepResolved)
+mixDepsCmdOutputParser = Map.fromList <$> (parseDep `sepBy` char '*') <* eof
   where
     findName :: Parser Text
     findName = takeWhileP (Just "dep") (/= ' ')
 
     findVersion :: Parser Text
-    findVersion = T.pack <$> some (alphaNumChar <|> char '.' <|> char '-')
+    findVersion = Text.pack <$> some (alphaNumChar <|> char '.' <|> char '-')
```

Comment:
[nit]: Use `toText` from `Data.String.Conversion` instead of `Text.pack`.
```suggestion
    findVersion = toText <$> some (alphaNumChar <|> char '.' <|> char '-')
```

Reply:
+style guide https://github.com/fossas/team-analysis/issues/671